portal/models/message.py

```diff
@@ -180,7 +180,7 @@ def send_message(self, cc_address=None):
         else:
             # This should never happen, alert if it does
             current_app.logger.error(
-                "Unable to generate audit log for email {}".format(Message))
+                "Unable to generate audit log for email: %s", str(Message))
```

Comment:
just curious, did you find that explicit cast to `str` necessary? I too thought it was automatically cast when you use string formatting